jenkins/opensearch-dashboards/integ-test.jenkinsfile

```diff
@@ -21,9 +21,9 @@ def docker_images = [
 ]
 
 def docker_args = [
-    "tar": "-u 1000 -e BROWSER_PATH=electron",
-    "rpm": "--entrypoint=/usr/lib/systemd/systemd -u root --privileged -v /sys/fs/cgroup:/sys/fs/cgroup:rw --cgroupns=host -e BROWSER_PATH=electron",
-    "deb": "--entrypoint=/usr/lib/systemd/systemd -u root --privileged -v /sys/fs/cgroup:/sys/fs/cgroup:rw --cgroupns=host -e BROWSER_PATH=electron",
```

Comment:
Are we running on Chromium then?

Reply:
Yes, we were supposed to remove these since this PR:
* https://github.com/opensearch-project/opensearch-dashboards-functional-test/pull/1123